README.md

```diff
@@ -15,6 +15,22 @@ To accomplish these tasks reliably and to offer a certain quality of service, Ga
 Please find more information regarding the concepts and a detailed description of the architecture in our [Gardener Wiki](https://github.com/gardener/documentation/wiki/Architecture) and our [blog post on kubernetes.io](https://kubernetes.io/blog/2018/05/17/gardener/).
 
 ----
+## K8s e2e conformance test coverage
```

Comment:
I'd just rephrase to `Kubernetes conformance tests coverage`
(The rest of the document uses Kubernetes as well instead of K8s and the fact that the conformance tests are a subset of the e2e test suite is probably not relevant in the heading. I think `conformance tests` stands on its own.)

Reply:
done